src/plugins/tooltip/c-tooltip.vue

```diff
@@ -2,24 +2,29 @@
   <component
     :is="rootElement"
     :class="b()"
-    @mouseenter="onMouseEnter"
-    @mouseleave="onMouseLeave"
+    @mouseenter="showTooltip"
+    @mouseleave="tooltipVisible = false"
   >
     <!-- @slot The slot for the tooltips anchor element(s). -->
     <slot></slot>
 
     <!-- Moves tooltip to a portal to prevent invalid HTML nesting. -->
-    <Teleport to="body">
-      <div
-        ref="tooltip"
-        :class="b('tooltip-wrapper')"
-        @load.capture="onLoad"
-      >
-        <div :class="b('tooltip')">
-          <!-- @slot This slot will be used for the tooltip content. -->
-          <slot name="tooltip"></slot>
+    <Teleport v-if="tooltipInitialized" to="body">
```

Comment:
Is using `v-if` directly on the Teleport the right approach? In a Vue 3 docs example they are applying it to the child container inside the Teleport.

https://vuejs.org/guide/built-ins/teleport.html